site/en/examples/doc_search_emb.ipynb

```diff
@@ -659,10 +659,12 @@
       ],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        embeddings.append(emb['embedding'])
</pre><p> </p><p>I think you can use <code>df.apply</code> here without losing any clarity or flexibility - e.g.</p><pre class="ql-syntax" spellcheck="false">def embed_fn(text):
  return palm.generate_embeddings(model=model, text=text)['embedding']

df['Embeddings'] = df['Text'].apply(embed_fn)
</pre><p> </p><p>WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>